Neuron/Models/Realm/LocalTxDetailModel.swift

```diff
@@ -0,0 +1,130 @@
+//
+//  SentTransaction.swift
+//  Neuron
+//
+//  Created by 晨风 on 2018/11/15.
+//  Copyright © 2018 Cryptape. All rights reserved.
+//
+
+import UIKit
+import RealmSwift
+import BigInt
+import AppChain
+
+class LocalTxDetailModel: Object {
+    @objc dynamic var token: TokenModel!
+    @objc dynamic var txHash = ""
+    @objc dynamic var blockNumber: Int = 0
+    @objc dynamic var from = ""
+    @objc dynamic var to = ""
+    @objc dynamic var valueText = "0"
+    @objc dynamic var gasPriceText = "0"
```

Comment:
It should be better to save gasPrice as text but name the field as `gasPrice` (and for `value`, `gasLimit`, etc.). Then convert `BigUInt` and `String` when initializing the model instance.

By doing that, the DB table will make sense as field should not have to specify their type with suffix.